e2e/utils/require.go

```diff
@@ -0,0 +1,45 @@
+package utils
+
+import (
+	"fmt"
+
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
+	"github.com/stretchr/testify/require"
+
+	crosschaintypes "github.com/zeta-chain/zetacore/x/crosschain/types"
+)
+
+// RequireCCTXStatus checks if the cctx status is equal to the expected status
+func RequireCCTXStatus(
+	t require.TestingT,
+	cctx *crosschaintypes.CrossChainTx,
+	expected crosschaintypes.CctxStatus,
+	msgAndArgs ...any,
+) {
+	msg := fmt.Sprintf("cctx status is not %q", expected.String())
+
+	require.NotNil(t, cctx.CctxStatus)
+	require.Equal(t, expected, cctx.CctxStatus.Status, msg+errSuffix(msgAndArgs...))
+}
+
+// RequireReceiptApproved checks if the receipt status is successful.
+func RequireReceiptApproved(t require.TestingT, receipt *ethtypes.Receipt, msgAndArgs ...any) {
```

Comment:
Maybe `RequireReceiptSuccessful` or `RequireTxSuccessful` is a more general name? It can be used for any transactions' receipt TBO.